certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/action/model/CertificateActionPrint.java

```diff
@@ -0,0 +1,29 @@
+package se.inera.intyg.certificateservice.domain.action.model;
+
+import java.util.List;
+import java.util.Optional;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import se.inera.intyg.certificateservice.domain.certificate.model.Certificate;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.CertificateActionSpecification;
+
+@Builder
+@Getter(AccessLevel.NONE)
+public class CertificateActionPrint implements CertificateAction {
```

Comment:
Need to implement name and description for the action. See for example CertificateActionDelete.